cli/src/stake.rs

```diff
@@ -379,6 +379,12 @@ impl StakeSubCommands for App<'_, '_> {
                         .help("Seed for address generation; if specified, the resulting account \
                                will be at a derived address of STAKE_ACCOUNT_ADDRESS")
                 )
+                .arg(
+                    Arg::with_name("delinquent")
```

Comment:
I wish we could do this without the parameter, but I guess it's too much to check the vote account for delinquency on every deactivation attempt...

Reply:
I also wanted this to be explicit by the user as a way to ensure the responsibly for performing the action is clearly with the user